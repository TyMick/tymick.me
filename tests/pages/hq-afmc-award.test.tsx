import React from "react";
import renderer from "react-test-renderer";
import HqAfmcAward from "../../pages/hq-afmc-award";

describe("HqAfmcAward", () => {
  it("matches snapshot", async () => {
    expect(renderer.create(<HqAfmcAward />).toJSON()).toMatchSnapshot();
  });
});
