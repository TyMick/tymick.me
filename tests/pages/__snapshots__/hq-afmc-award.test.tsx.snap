// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HqAfmcAward matches snapshot 1`] = `
<div
  className="cap-width-lg container"
>
  <h1>
    2014 HQ AFMC Category I Civilian of the Year
  </h1>
  <div
    className="Image__ImageWrapper-sc-wuromz-0 cBHqVO mb-3 mx-md-n3 full-width"
  >
    <span
      style={
        {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
          style={
            {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt="Ty Mick, wearing a medal featuring the AFMC logo, walking on a red carpet between Airmen holding swords at the 2014 AFMC Annual Excellents Awards."
        className=""
        data-nimg="intrinsic"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="/_next/image?url=%2Fimg.jpg&w=48&q=75"
        srcSet="/_next/image?url=%2Fimg.jpg&w=32&q=75 1x, /_next/image?url=%2Fimg.jpg&w=48&q=75 2x"
        style={
          {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
    </span>
  </div>
  <p>
    In my time working in the Studies and Analyses Division of Headquarters Air Force Materiel Command, I had the honor of being named the 2014 HQ AFMC Category I Civilian of the Year, which gave me the opportunity to represent the Headquarters at AFMCâ€™s Annual Excellence Awards banquet, pictured above.
  </p>
</div>
`;
